pkgs/servers/sql/postgresql/ext/pgsql_http.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, curl, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "pgsql_http";
+  version = "v1.3.1";
+
+  buildInputs = [ curl postgresql ];
+
+ src = fetchGit {
+    url = "git://github.com/pramsey/pgsql-http";
```

Comment:
Please use fetchFromGitHub.

Reply:
why? fetchGit is a builtin, what's wrong with using it?